src/sentry/snuba/metrics/naming_layer/mapping.py

```diff
@@ -103,6 +106,12 @@ def extract_custom_metric_alias(internal_name: str) -> Optional[str]:
         and match.group("namespace") == "transactions"
     ):
         return match.group("name")
+    elif (
+        match is not None
+        and match.group("entity") == "c"
+        and match.group("namespace") == "escalating_issues"
+    ):
+        return match.group("name")
```

Comment:
Even though this is functionally correct, it's semantically wrong. This function has the only purpose of extracting aliases for custom metrics that do not have hardcoded MRIs in the enums.

You would have to add your `ErrorsMRI` and `ErrrosMetricKey` here (https://github.com/getsentry/sentry/blob/1bd878da221af076363a47aa0c25f14f4e5c4842/src/sentry/snuba/metrics/naming_layer/mapping.py#L43-L43) for a proper integration and this will result in this (https://github.com/getsentry/sentry/blob/1bd878da221af076363a47aa0c25f14f4e5c4842/src/sentry/snuba/metrics/naming_layer/mapping.py#L83-L83) succeeding.